main.nf

```diff
@@ -595,10 +595,10 @@ if(!params.keep_phix) {
             saveAs: {filename -> filename.indexOf(".fastq.gz") == -1 ? "QC_shortreads/remove_phix/$filename" : null}
 
         input:
-        set val(name), file(reads), file(genome), file(db) from trimmed_reads.combine(phix_db)
+        set val(name), file(reads), file(genome), file(db) from ch_trimmed_reads.combine(ch_phix_db)
 
         output:
-        set val(name), file("*.fastq.gz") into (trimmed_reads_megahit, trimmed_reads_metabat, trimmed_reads_fastqc, trimmed_sr_spadeshybrid, trimmed_reads_spades, trimmed_reads_centrifuge, trimmed_reads_kraken2, trimmed_reads_bowtie2, trimmed_reads_filtlong)
+        set val(name), file("*.fastq.gz") into (ch_reads_megahit, ch_trimmed_reads_metabat, ch_trimmed_reads_fastqc, ch_short_reads_spadeshybrid, ch_reads_spades, ch_reads_centrifuge, ch_trimmed_reads_kraken2, ch_reads_bowtie2, ch_short_reads_filtlong)
```

Comment:
Why sometimes with "trimmed" and sometimes without? E.g. `ch_reads_centrifuge`,`ch_reads_megahit` but `ch_trimmed_reads_kraken2`,`ch_trimmed_reads_metabat`?

It does make sense for fastqc, you have currently `ch_raw_reads_fastqc` and `ch_trimmed_reads_fastqc`. But also here, for the sake of standardising, "trimmed" could be omitted and only "raw" could be used if its not trimmed.

Also, it might be actually good to always add "short" or "long" for reads, might make it even more readable and easier to follow.